hack/rebuild-local-kind-with-quay.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+set -euo pipefail
+
+# This can be used to destroy and rebuild a working cluster locally with
+# kind and a quay.io registry. Expects an .env file to exist and to define
+# some required secrets, including credentials for GitHub and Quay.
+
+function slow-title() {
+  sleep 2
+  echo ""
+  echo "************************************************************"
+  echo " 🐢 $1"
+  echo "************************************************************"
+  echo ""
+}
+
+slow-title "Deleting old cluster"
```

Comment:
All the the deploy scripts are idempotent , thus this script should be as well. Do you think the kind cluster should be always recreated? I think it should be optional. 

Another point, when using the podman provider for kind there is another step to make it work (you can see it in the Readme)